Restore functionality from 3.99.5final.SP08 so that cyberJack secoder still
works.

Index: cjeca32/USBUnix.cpp
--- cjeca32/USBUnix.cpp.orig
+++ cjeca32/USBUnix.cpp
@@ -288,6 +288,17 @@ int CUSBUnix::Open() {
     m_intPipe=0x81;
     break;
 
+  case 0x400:
+    Debug.Out("<USB>",
+	      DEBUG_MASK_COMMUNICATION_ERROR,
+	      "Using USB implementation 3", 0, 0);
+    usbMode=3;
+
+    m_bulkOut=0x02;
+    m_bulkIn=0x81;
+    m_intPipe=0x83;
+    break;
+
   case 0x401:
     Debug.Out("<USB>",
 	      DEBUG_MASK_COMMUNICATION_ERROR,
